app/views/meditations/index.html.slim

```diff
@@ -1,99 +1,80 @@
 
 - content_for :title, @static_page.title
 
-.container
-    .breadcrumbs test
-
-section.with.half.left.blue.gradient.page-title
+section.intro
   .container
-    /h1.title = Meditation.model_name.human(count: -1)
-    h1.title Meditate Now
+    h1.title = @static_page.title
```

Comment:
The format used for the header here is common to several other pages, so we use the `section.intro` tag to consistently style them all.